Remotion/Web/ClientScript/Scripts/Utilities.ts

```diff
@@ -396,21 +406,40 @@ class LayoutUtility
     // complexity for backwards compatibility that we don't need (IE).
     const value = Math.round(parseFloat(style[dimension])) || 0;
     
-    // For content-box: value is already correct
-    // For border-box: we need to remove padding and border from the value
-    if (style.boxSizing !== "border-box")
-      return value;
-    
     const orientations: CssOrientation[] = dimension === "width"
       ? ["Left", "Right"]
       : ["Top", "Bottom"];
 
+    // For content-box: value is already the width/height
+    // For border-box: we need to remove padding/border
+    if (style.boxSizing !== "border-box")
+    {
+      if (extra === undefined)
+        return value;
+
+      let adjustment = 0.0;
+      for (const orientation of orientations)
+      {
+        if (extra === "padding")
```

Comment:
Am I missing the border?

Reply:
No because there are already methods that return the size with border and padding, here we just want the padding.